python/semantic_kernel/skill_definition/sk_function_decorator.py

```diff
@@ -1,17 +1,23 @@
 # Copyright (c) Microsoft. All rights reserved.
 
 
-def sk_function(description: str):
+def sk_function(description: str, name: str = None, input_description: str = None, input_default_value: str = None):
```

Comment:
Do we want to force users to use keyword args with this decorator? Might be nice for consistency/style? Open to your thoughts either way!

Reply:
I was wondering about that, might be better for consistency, I agree.